platform/linux-generic/include-abi/odp/api/abi/pool.h

```diff
@@ -17,23 +18,7 @@
 extern "C" {
 #endif
 
-#include <odp/api/std_types.h>
-#include <odp/api/plat/strong_types.h>
-#include <odp/api/abi/event.h>
-
-/** @ingroup odp_pool
- *  @{
- */
-
-typedef ODP_HANDLE_T(odp_pool_t);
-
-#define ODP_POOL_INVALID _odp_cast_scalar(odp_pool_t, 0)
-
-#define ODP_POOL_NAME_LEN  32
-
-/**
- * @}
- */
+/* Empty placeholder header for inline functions */
```

Comment:
Nitpick: some empty header files comment `/* Empty header required due to the packet inline functions */` and some `/* Empty placeholder header for inline functions */`, for different purpose or just use the same in all?

Reply:
There is a small difference. In ABI compatible mode `pool.h` header is picked from `include/odp/api/abi-default/` and this header will remain empty also in the future. In non-ABI compatible mode `platform/linux-generic/include-abi/odp/api/abi/pool.h` is used instead. When we want to inline pool functions, the inline header will be included using this file (e.g. `platform/linux-generic/include-abi/odp/api/abi/queue.h`).